sys-process/nmon/nmon-16i.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic toolchain-funcs
+
+MY_P="lmon${PV}"
+
+DESCRIPTION="Nigel's performance MONitor for CPU, memory, network, disks, etc..."
+HOMEPAGE="http://nmon.sourceforge.net/"
+LICENSE="GPL-3"
+SRC_URI="mirror://sourceforge/${PN}/${MY_P}.c"
+
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc64 ~x86"
+
+RDEPEND="sys-libs/ncurses:0="
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+"
+
+S="${WORKDIR}"
+
+src_unpack() {
+	cp -v -f "${DISTDIR}"/${MY_P}.c "${S}"/${PN}.c || die
```

Comment:
`-f` makes `|| die` rather useless. Please remove `-f` and (personal opinion) also `-v`.